src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax;
+
+import java.math.RoundingMode;
+import java.text.DecimalFormat;
+
+public class Circle extends Figure {
+    private final double radius;
```

Comment:
radius expected to be int
```suggestion
    private final int radius;
```